pkg/commands/products/root.go

```diff
@@ -0,0 +1,189 @@
+package products
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/fastly/go-fastly/v8/fastly"
+
+	"github.com/fastly/cli/pkg/cmd"
+	fsterr "github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/global"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+)
+
+// RootCommand is the parent command for all subcommands in this package.
+// It should be installed under the primary root command.
+type RootCommand struct {
+	cmd.Base
+	cmd.JSONOutput
+
+	disableProduct string
+	enableProduct  string
+	manifest       manifest.Data
+	serviceName    cmd.OptionalServiceNameID
+}
+
+// ProductEnablementOptions is a list of products that can be enabled/disabled.
+var ProductEnablementOptions = []string{
+	"brotli_compression",
+	"domain_inspector",
+	"fanout",
+	"image_optimizer",
+	"origin_inspector",
+	"websockets",
+}
+
+// NewRootCommand returns a new command registered in the parent.
+func NewRootCommand(parent cmd.Registerer, g *global.Data, m manifest.Data) *RootCommand {
+	var c RootCommand
+	c.Globals = g
+	c.manifest = m
+	c.CmdClause = parent.Command("products", "Enable, disable, and check the enablement status of products on your services")
```

Comment:
```suggestion
	c.CmdClause = parent.Command("products", "Enable, disable, and check the enablement status of products")
```

Not sure this is the best suggestion, but I think that dropping _your services_ would be consistent with other command descriptions.